splitio/engine/evaluator.py

```diff
@@ -128,24 +128,24 @@ def evaluate_features(self, features, matching_key, bucketing_key, attributes=No
         :param attributes: An optional dictionary of attributes
         :type attributes: dict
 
-        :return: The treatments for the key and splits
+        :return: The treatments for the key and feature flags
         :rtype: object
         """
         return {
-            feature: self._evaluate_treatment(feature, matching_key,
-                                              bucketing_key, attributes, split)
-            for (feature, split) in self._split_storage.fetch_many(features).items()
+            feature_flag_name: self._evaluate_treatment(feature_flag_name, matching_key,
+                                              bucketing_key, attributes, feature_flag)
+            for (feature_flag_name, feature_flag) in self._feature_flag_storage.fetch_many(feature_flag_names).items()
         }
 
-    def _get_treatment_for_split(self, split, matching_key, bucketing_key, attributes=None):
+    def _get_treatment_for_split(self, feature_flag, matching_key, bucketing_key, attributes=None):
         """
         Evaluate the feature considering the conditions.
 
         If there is a match, it will return the condition and the label.
         Otherwise, it will return (None, None)
 
-        :param split: The split for which to get the treatment
-        :type split: Split
+        :param feature_flag: The feature flag for which to get the treatment
+        :type feature_flag: Split
```

Comment:
Double checking, is `Split` a type or should it be changed?

Reply:
It is a class, I put the task to change it in "Could" list